specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -4343,6 +4343,14 @@
         "enableMultipleWriteLocations": {
           "description": "Enables the account to write in multiple locations",
           "type": "boolean"
+        },
+        "enableCassandraConnector": {
```

Comment:
This involves adding a mutable property to an existing API version, which is considered a form of breaking change as it breaks the "GET->PUT pipeline." However, in this case, I think it is fairly safe.